apps/api/.env.example

```
@@ -1,2 +1,4 @@
+AUTH_BASE_URL="http//localhost:3001"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)